torch/onnx/_internal/exporter.py

```diff
@@ -153,21 +194,39 @@ def save(
 
 
 class Exporter(abc.ABC):
+    class WrappedFuncModule(torch.nn.Module):
+        def __init__(self, forward: Callable):
+            super().__init__()
+            self.actual_forward = forward
+
+        def forward(self, *args, **kwargs):
+            result, _ = _pytree.tree_flatten(self.actual_forward(*args, **kwargs))
+            return result
+
```

Comment:
this wasnt discussed during design and I feel not 100% needed for the vision we have. This is a `FX Extractor` specific task, limited to the FX APIs that do not support `**kwargs`.

Reply:
It is needed to support the tests we have authored so far. This should change as we continue to refactor.